chart/templates/post-delete-hook-clusterRole.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.postDelete.enabled }}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Values.postDelete.name }}
```

Comment:
Why does the user need to provide `.postDelete.name`? Let's use something like `{{ template "rancher.fullname" . }}-post-delete` instead if possible and remove the provided name entirely from the values.yaml.

Reply:
same comment for everywhere else this is used.

Reply:
fixed.